test/menu-route-test.js

```diff
@@ -0,0 +1 @@
+'use strict';
```

Comment:
I would let Kyler add this file. No need for you to upload the first version with only 'use strict';